### DISCUSSION TITLE
Status check

### DISCUSSION THREAD

**Discussion by Cooper-Kunz** [hashgraph/hedera-sdk-swift#29] [General]
Hello, what is the status of this SDK? 

Is it ready for usage, under active development (hasn't been contributed to in a month, so was curious), abandoned, etc.? 

I know of multiple projects that would use this if/when it's ready & it's already been added to [docs.hedera.com](https://docs.hedera.com/guides/docs/sdks)

**Comment by izik1** [comment]
Hi, sorry for not replying to this, stuff in the discussions tab doesn't show up for me anywhere I'd get a notification/email.

This is under active development, it is is a pre-production state currently (although far enough along to use it), mostly work right now is centered around adding tests and documentation to prepare for a 1.0 release.

At the time I believe this was when the Swift SDK (this one) was still in the Rust SDK Repo for build-related versions, and this repo only existed for publishing releases, hence the low activity at the time. However that has changed and the SDK is now completely standalone.

Hope this helps!
